d.ts/eclint.d.ts

```diff
@@ -2,6 +2,8 @@
 /// <reference path="../typings/gulp-util/gulp-util.d.ts" />
 /// <reference path="../node_modules/linez/linez.d.ts" />
```

Comment:
If you upgrade `linez` to the latest version and change the import to `import linez from 'linez';` you shouldn't need this `<reference` any more.